xfce-base/xfwm4/xfwm4-4.16.0.ebuild

```diff
@@ -29,6 +29,7 @@ RDEPEND=">=dev-libs/glib-2.20
 	>=xfce-base/xfconf-4.13:=
 	opengl? ( media-libs/libepoxy:=[X(+)] )
 	startup-notification? ( x11-libs/startup-notification )
+	xi? ( x11-libs/libXi )
```

Comment:
Is there actually much point in having it conditional? Maybe enable it unconditionally. It seems that a lot of packages require it on my system. The same probably goes for xcomposite.

Reply:
No, I just did it to be consistent with the other options.  Updated.